be1-go/channel/election/mod.go

```diff
@@ -500,13 +503,39 @@ func checkMethodProperties(method string, length int) error {
 	return nil
 }
 
-func changeVote(qs *question, earlierVote validVote, sender string, created int64, indexes []int) {
-	if earlierVote.voteTime > created {
-		qs.validVotes[sender] = validVote{
-			voteTime: created,
-			indexes:  indexes,
+func updateVote(msgID string, sender string, castVote messagedata.VoteCastVote, questions *map[string]*question) error {
```

Comment:
maps are reference types and always passed by reference. So you shouldn't need to pass a pointer of map.